test/unit/src/common/actions/t_snaps.js

```diff
@@ -21,10 +22,12 @@ const mockStore = configureMockStore(middlewares);
 
 describe('repositories actions', () => {
   const initialState = {
-    isFetching: false,
-    success: false,
-    error: null,
-    snaps: null
+    snaps: {
```

Comment:
Created a card for that: https://trello.com/c/nnpyx939/320-actions-tests-use-wrong-initialstate